.github/container/nsys_jax/nsys_jax/analyses/communication.py

```diff
@@ -31,14 +49,43 @@ def process_communication_data(steady_state):
         # therefore the last one to join the collective and its bandwidth estimate does
         # not contain a wait time component. The .mean() is over the different
         # (ProgramId, ProgramExecution, ThunkIndex) values.
-        bandwidth = devices["BusBandwidthGBPerSec"].agg("max")
+        max_bandwidth_per_device = devices["BusBandwidthGBPerSec"].agg("max")
```

Comment:
The `max` is over devices, so `max_bandwidth_per_device` is misleading. It's more like `bandwidth_of_fastest_device`.